acvm_backend_barretenberg/src/acvm_interop/proof_system.rs

```diff
@@ -62,12 +60,16 @@ impl ProofSystemCompiler for Plonk {
         circuit: &Circuit,
         verification_key: &[u8],
     ) -> bool {
-        let mut composer = StandardComposer::new(circuit.into());
-
         // Unlike when proving, we omit any unassigned witnesses.
         // Witness values should be ordered by their index but we skip over any indices without an assignment.
         let flattened_public_inputs: Vec<FieldElement> = public_inputs.into_values().collect();
 
-        composer.verify_with_vk(proof, flattened_public_inputs.into(), verification_key)
+        Composer::verify_with_vk(
```

Comment:
This is due to name collisions with the `Backend` trait, Rust can't figure out which method we want to call so we need to be explicit.

Reply:
I'm happy to change the internal name, I've not been a fan of them having conflicting names. It probably should be `verify_constraint_system_with_vk` since it takes the CS instead of Circuit.